keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -2547,7 +2547,7 @@
           "type": "array",
           "items": {
             "type": "string",
-            "description": "JSON web key operations. For more information, see JsonWebKeyOperation.",
+            "description": "JSON web key operations. The default is all operations. For more information, see JsonWebKeyOperation.",
```

Comment:
Must be a change since the version I documented on MSDN for the October release of the same content. Oversight by the product team or my mistake. I'll have to check this out with my dev contact. 

@pomortaz - Pooneh, do you know if ALL is a default for this as it was before?